alerts/openshift-virtualization-operator/NetworkAddonsConfigNotReady.md

```diff
@@ -30,7 +30,7 @@ deployment or daemon set that is not ready:
 2. Check the component's pod for errors:
 
    ```bash
-   $ oc -n cluster-network-addons get daemonset <pod> -o yaml
+   $ oc -n cluster-network-addons get daemonset <demonset> -o yaml
```

Comment:
there is no such namespace `cluster-network-addons`. It should be in the same namespace as kubevirt.
also, typo in `demonset` (should be `daemonset`).